Content.Client/Chat/TypingIndicator/TypingIndicatorVisualizerSystem.cs

```diff
@@ -9,42 +9,28 @@ public sealed class TypingIndicatorVisualizerSystem : VisualizerSystem<TypingInd
 {
     [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
 
-    public override void Initialize()
+    protected override void OnAppearanceChange(EntityUid uid, TypingIndicatorComponent component, ref AppearanceChangeEvent args)
     {
-        base.Initialize();
-        SubscribeLocalEvent<TypingIndicatorComponent, ComponentInit>(OnInit);
-    }
+        base.OnAppearanceChange(uid, component, ref args);
 
-    private void OnInit(EntityUid uid, TypingIndicatorComponent component, ComponentInit args)
-    {
         if (!TryComp(uid, out SpriteComponent? sprite))
             return;
```

Comment:
AppearanceChangeEvent already has a reference to spritecomp I think.

Reply:
Yep, done